### ISSUE TITLE
Could anyone please share their MS DIA methods or settings here?

### ISSUE THREAD

**Issue by chingciripit** [vdemichev/DiaNN#875]
Hi,

I am currently using Thermo Scientific Q Exactive mass spectrometer using ESI ion positive mode with a Phenomenex Kinetex 2.6 μm XB-C18 100 Å (150 × 4.6 mm) column. I can't seem to get meaningful data from my samples.

Thank you.

Ching

**Comment by michaelsteidel86** [comment]
- Are you refering to MS instrument method settings or DIA-NN parameters?
- Would you mind providing more details on sample type?
- Please share the a relevant log file.

**Comment by michaelsteidel86** [comment]
There various published ones, i.e.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7124470/
Optimal settings mainly depend on sample type and gradient length.

Would you mind sharing details on your MS settings?

**Comment by chingciripit** [comment]
Yes, please see attached. I was wondering if Q Exactive Orbitrap was suitable for DIA analysis? I can select the DIA analysis on the MS instrument settings. However, many papers that I've read seem to describe Orbitrap Exploris 480, not Q Exactive Orbitrap.

<img width="1634" alt="Full MS" src="https://github.com/vdemichev/DiaNN/assets/152452696/aa089177-7b93-4164-b97c-c817424f18b5">
<img width="1563" alt="DIA" src="https://github.com/vdemichev/DiaNN/assets/152452696/ec45a846-d832-4fe1-910c-a25882ac562c">

Thanks!
